definitions/npm/bser_v1.x.x/flow_v0.22.x-/bser_v1.x.x.js

```diff
@@ -0,0 +1,58 @@
+declare type $npm$bser$BserTypes =
```

Comment:
As soon as you declare at least one `declare` statement as `declare export ...`, the module will be treated as an ES6 module and will not automatically export everything... so if you put a `declare type BserTypes = ...` inside the `declare module` scope and export types etc. as `declare export class Acumulator` etc., it should work.

Example:
```
declare module 'bser' {
   declare type BserTypes = {...};

   declare export class Accumulator { ... };
   declare export function loadFromBuffer(......);
}
```
And so on... the `declare export` syntax is only supported by more modern versions of flow (I guess >= 0.30 or so... you need to try that).

Hope this helps